Automattic/wp-calypso
# Help Center: Close help center when notifications panel is opened

Fixes: https://github.com/Automattic/wp-calypso/issues/68781

## Proposed Changes

* Add changes to close Help Center when the notifications panel is opened.

## Why are these changes being made?


* The notifications panel is closed if the Help Center is opened. We want the same behavior when the Help Center is opened, which would close the notifications panel.

## Testing Instructions



* Checkout branch or use the live link to test Calypso changes
* Sync `apps/help-center` to WPCOM. Sandbox your site and widget API.
* Open the Help Center from the masterbar then open the notifications panel.
* Verify the Help Center is closed.

## Pre-merge Checklist
